docs/tSNE_dimensionality_reduction.md

```diff
@@ -101,45 +101,46 @@ Depending on your access level in R2, you can create t-SNE maps from any dataset
 
 Let's take a look at some other nice example of a R2 generated t-SNE maps: the large dataset of normal tissue expression profiles
 
-1. In main menu select  Normal tissues - GTEX - 2921 - RPKM - ensgtexv4 en box2 and select t-SNE. If the 'default' map has already been calculated, a shortcut button will also appear as shown by the dashed box in figure 7. Under he 'Adjustable settings' you can adjust severall settings such as sample filtering, specific gene categories and expression level restrictions.  
+1. In main menu select  Normal Tissues GTeX v4 - GTeX - 2921 - RPKM - ensgtexv4 inn box2 and select t-SNE in box3. Click Next. If the 'default' map has already been calculated, a shortcut button will also appear as shown by the dashed box in figure 7. Under the 'Adjustable settings' you can adjust several settings such as sample filtering, specific gene categories and expression level restrictions.  
 
 	![Figure   7: t-SNE: Menu ](_static/images/Tnse_shortcutPlot.png "Figure 7: All perplexities")
 	
 	[**Figure 7: t-SNE: Menu**](_static/images/Tnse_shortcutPlot.png)
 
-Keep in mind that after adjusting input settings the t-SNE algorithm will  re-run again although a t-SNE map already has been generated with the default settings. A note on the execution times of t-SNE: the generation of the maps will take a substantial amount of time to generate, especially for larger datasets (up to a number of hours for datasets >600 samples). Once initiated (showing the message that t-SNE is being calculated), you can close the window and return to the analysis at a later time. The process will keep on running in the background. 
+Keep in mind that after adjusting input settings the t-SNE algorithm will  re-run again,  even though a t-SNE map already has been generated with the default settings. A note on the execution times of t-SNE: the generation of the maps will take a substantial amount of time to generate, especially for larger datasets (up to a number of hours for datasets >600 samples). Once initiated (showing the message that t-SNE is being calculated), you can close the window and return to the analysis at a later time. The process will keep on running in the background. 
```

Comment:
the process will keep on running, but how will the user be notified? (not, I presume but make that clear?)